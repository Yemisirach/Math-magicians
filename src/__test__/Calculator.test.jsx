import React from "react";
import renderer from "react-test-renderer";
import Calculator from "../Components/Calculator";

describe("Calculator component", () => {
  it("should render correctly and match the saved snapshot", () => {
    const tree = renderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
