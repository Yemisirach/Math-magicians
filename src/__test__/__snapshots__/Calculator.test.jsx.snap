// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component should render correctly and match the saved snapshot 1`] = `
<div>
  <div
    className="mainCalculator"
  >
    <div>
      <h2>
        Les's do some math
      </h2>
    </div>
    <div
      className="allCal"
    >
      <div
        className="calculatorOutput"
      >
        <div
          className="CalcInput"
        >
          0
        </div>
      </div>
      <div
        className="Calc_btns"
      >
        <div
          className="button-col"
        >
          <button
            name="AC"
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            name="7"
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            name="4"
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            name="1"
            onClick={[Function]}
            type="button"
          >
            1
          </button>
        </div>
        <div
          className="button-col"
        >
          <button
            name="+/-"
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            name="8"
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            name="5"
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            name="2"
            onClick={[Function]}
            type="button"
          >
            2
          </button>
        </div>
        <div
          className="button-col"
        >
          <button
            name="%"
            onClick={[Function]}
            type="button"
          >
            %
          </button>
          <button
            name="9"
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            name="6"
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            name="3"
            onClick={[Function]}
            type="button"
          >
            3
          </button>
        </div>
        <div
          className="button-col last-col"
        >
          <button
            name="รท"
            onClick={[Function]}
            type="button"
          >
            รท
          </button>
          <button
            name="x"
            onClick={[Function]}
            type="button"
          >
            x
          </button>
          <button
            name="-"
            onClick={[Function]}
            type="button"
          >
            -
          </button>
          <button
            name="+"
            onClick={[Function]}
            type="button"
          >
            +
          </button>
        </div>
      </div>
      <div
        className="button-row"
      >
        <button
          name="0"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <div
          className="pointrow"
        >
          <button
            name="."
            onClick={[Function]}
            type="button"
          >
            .
          </button>
          <div
            className="last-col"
          >
            <button
              name="="
              onClick={[Function]}
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
